WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsV4PeriodViewController.swift

```diff
@@ -655,8 +706,26 @@ private extension StoreStatsV4PeriodViewController {
         reloadSiteFields()
         reloadChart(animateChart: animateChart)
         reloadLastUpdatedField()
-        let visitStatsElements = shouldShowSiteVisitStats ? [visitorsTitle as Any,
-                                                             visitorsData as Any]: []
+        let visitStatsElements: [Any] = {
+            switch siteVisitStatsMode {
+            case .default:
+                // TODO-5450: Temporary solution to make the empty view show up.
+                // Remove this when JCP site is listed and can be selected.
+                if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.jetpackConnectionPackageSupport) {
```

Comment:
I'm a bit confused by why feature flag is needed here 🤔 in the `default` case where site visit stats are shown and the empty view is hidden, why not always showing `[visitorsTitle as Any, visitorsData as Any]`?

Reply:
This was intended for testing while the implementation for loading JCP sites wasn't available yet. But now that #5466 was merged, I have replaced these checks and updated the display logic in 7784b1283f49d0577706714f5d64c1ba0a33a3a4.